src/bacnet/basic/object/csv.c

```diff
@@ -148,13 +148,13 @@ unsigned CharacterString_Value_Instance_To_Index(uint32_t object_instance)
 uint32_t CharacterString_Value_Index_To_Instance(unsigned index)
 {
 
-    if(index < CSV_Max_Index){
+    if(index < CSV_Max_Index) {
         return CSV_Descr[index].Instance;
     } else {
-        PRINT("index out of bounds %d", CSV_Descr[index].Instance);
+       PRINT("index out of bounds");
```

Comment:
Would this not be useful debug to keep?

Reply:
Same as above the other comment, I'm correcting them